### PR TITLE
chore: standardise api keys

### PR DESCRIPTION
**Pull request by dabeeeenster**

### CODE REVIEW

File: docs/docs/clients/server-side.md [hunk 1]
```diff
@@ -1824,7 +1824,7 @@ router.get('/', function (req, res, next) {
 <TabItem value="php" label="PHP">
 
 ```php
-$flagsmith = (new Flagsmith(TOKEN));
+$flagsmith = (new Flagsmith(FLAGSMITH_SERVER_SIDE_ENVIRONMENT_KEY));
```
Comment (zachaysan) [inline docs/docs/clients/server-side.md]: Shouldn't this be in quotes like the rest of the listings of the keys?

File: docs/docs/clients/server-side.md [hunk 2]
```diff
@@ -1836,7 +1836,7 @@ composer require symfony/cache
 ```
 
 ```php
-$flagsmith = (new Flagsmith(TOKEN))
+$flagsmith = (new Flagsmith(FLAGSMITH_SERVER_SIDE_ENVIRONMENT_KEY))
```
Comment (zachaysan) [inline docs/docs/clients/server-side.md]: Here too

**Review by zachaysan — APPROVED** [review]
One small non-showstopper question.

File: docs/docs/clients/server-side.md (hunk 1 above)
Comment (dabeeeenster) [reply docs/docs/clients/server-side.md]: Ooh well spotted

File: docs/docs/clients/server-side.md (hunk 2 above)
Comment (dabeeeenster) [reply docs/docs/clients/server-side.md]: YES

**Pull request merged**